components/event-footer/EventFooter.vue

```diff
@@ -31,7 +31,13 @@
 
 <style>
   .event-footer {
-    background: linear-gradient(rgba(0,0,0,0.7), rgba(0,0,0,0.9));
+    margin-top: 3rem;
```

Comment:
Should margin top be used? I try to avoid it since it can cause CSS margin collapsing. Instead we could put margin bottom on the component above the footer. Thoughts?

Reply:
Yes, that would be better indeed. 👍